klib/radar.c

```diff
@@ -1,13 +1,14 @@
 #include <kernel.h>
 #include <http.h>
+#include <log.h>
 #include <lwip.h>
 
 #define RADAR_HOSTNAME  "radar.relayered.net"
 
-#define RADAR_IP_1  34
-#define RADAR_IP_2  94
-#define RADAR_IP_3  88
-#define RADAR_IP_4  162
+#define RADAR_IP_1  35
```

Comment:
can we replace this w/just the host lookup? this is going to change quite a lot

Reply:
OK, I will enable DNS support in lwIP and use that to do a host lookup